pkg/kwokctl/cmd/export/logs/logs.go

```diff
@@ -70,7 +70,7 @@ func runE(ctx context.Context, flags *flagpole, args []string) error {
 	// get the optional directory argument, or create a tempdir under the kwok default working directory
 	var dir string
 	if len(args) == 0 {
-		t, err := file.TempDir(workdir, "")
+		t, err := os.MkdirTemp(workdir, "tmp")
```

Comment:
The path is `~/.kwok/clusters/kwok/tmp1408029719/kwok-kwok`, change to `~/.kwok/clusters/kwok/tmp/log1408029719`